CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ v1.8.0 [unreleased]
 
 ### Bugfixes
 
+-   [#20100](https://github.com/influxdata/influxdb/pull/20100): fix(write): Successful writes increment write error statistics incorrectly
```

Comment:
The changelog on `master-1.x` isn't currently being maintained. We usually update it on the release branch.

Reply:
So, should I remove that entry?  I updated both changelogs.